Caltech-IPAC/firefly
# Firefly-1037, Firefly-1038: multiple issues

#### [Firefly-1037](https://jira.ipac.caltech.edu/browse/FIREFLY-1037), [Firefly-1038](https://jira.ipac.caltech.edu/browse/FIREFLY-1038): Multiple Issues

 - Firefly-1037: added ipac and caltech logo to version dialog
 - Firefly-1038: fixed stretch so all planes of the cube are changed
 - Other: show checkbox on the stretch dropdown if that stretch is selected
 - Other: improved the heuristic to determine is images are noisy


#### Testing
   - https://fireflydev.ipac.caltech.edu/firefly-1037-1038/firefly/
   - Look at help dialog
   - Load a cube, change the stretch using the dropdown, the stretch should be the same on all planes
       - confirm by looking at the stretch on the dropdown, it should show a checkbox
